deployment/tf/modules/yc/logs/configs/parsers.conf

```diff
@@ -1,7 +1,12 @@
 [PARSER]
     NAME        lzy-log
     Format      regex
-    Regex       ^[^ ]*\s*[^ ]*\s*\w*\s*(?<time>[^ ]*\s*[^ ]*)\s*\[(?<thread>[^\]]+)\]\s*(?<level>[^ ]+)\s*(?<logger>[^ ]+)[^-]*-\s*(?<message>.*)
+    Regex       ^[^ ]*\s*[^ ]*\s*\w*\s*(?<time>[^ ]*\s*[^ ]*)\s*\[(?<thread>[^\]]+)\]\s*(?<level>[^ ]+)\s*(?<logger>[^ ]+)[^-]*-\s*(?<log>.*)
+
+[PARSER]
+    NAME        docker-fmt
+    Format      regex
+    Regex       ^(?<time>[^ ]*) (?<stream>stdout|stderr) (\w) (?<log>.*)
```

Comment:
For some reason docker provides timestamp and stream not as json key, but as a part of the string. Here we need to parse this string first to extract stream as json key.